src/lib/components/event/event-summary-row.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import { formatDate } from '$lib/utilities/format-date';
+
+  import EventDetails from './event-details.svelte';
+  import EventSingleDetail from './event-single-detail.svelte';
+
+  export let event: HistoryEventWithId;
+  export let expanded: boolean = false;
+
+  $: collapsed = expanded ? false : true;
+
+  const onLinkClick = () => {
+    if (!expanded) {
```

Comment:
It feels like having two variables (e.g. `expanded` and `collapsed`) could be confusing. Maybe we just stick with `expanded`?